people/development

```diff
@@ -0,0 +1,30 @@
+# Personal and professional development
+
+## Core skills
+As our team grows, we’re collectively identifying skills needed for each role. All team members should prioritize learning core skills, with each others’ support.
+2i2c will cover the cost of these opportunities and expect the time required to be part of team members’ standard work days. Learning that’s essential to our daily work is part of our daily work. 
+
+
+## Coaching
+Many 2i2c team members have had great experiences working with a coach. Having an external partner in your journey can help you reflect, explore and creatively realize your potential.
+All team members are encouraged to try working with a coach: [guidance on that here] (https://docs.google.com/presentation/d/1r5Sb4BmBgZM3Gyz4gLHfv2kxk6vFfmz2tczrMAqihLg/edit?usp=sharing). 
```

Comment:
FYI, I think this Google Doc link has edit access, is that intentional?

Reply:
Also, for usability/accessibility purposes it's good to avoid generic phrases like "here" in link hypertext https://usability.yale.edu/web-accessibility/articles/links

Reply:
Thank you for the usability reminder! Yes, everyone inside 2i2c should be able to edit the coaching guide to add recommendation, and the only reason not to make it viewable by others is that it lists people who've coached team members (which isn't particularly private).

Reply:
I see Chris has already merged, but maybe a callout explaining that the doc is intentionally editable by 2i2c staff and provide encouragement to submit edits/improvements/recommendations would help?